src/transformers/models/big_bird/configuration_big_bird.py

```diff
@@ -107,7 +107,7 @@ def __init__(
         num_hidden_layers=12,
         num_attention_heads=12,
         intermediate_size=3072,
-        hidden_act="gelu_fast",
+        hidden_act="gelu_new",
```

Comment:
@sgugger @LysandreJik -> this was actually an error. It should have never been `gelu_fast` since no model uses `gelu_fast`, they all use `gelu_new` as a default. Since those activations are identical, I think the change is correct here. Also there is no `gelu_fast` in Flax/JAX